Hyperbolic_triangulation_2/demo/Hyperbolic_triangulation_2/CMakeLists.txt

```
@@ -7,37 +7,31 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 # Instruct CMake to run moc automatically when needed.
 set(CMAKE_AUTOMOC ON)
 
-if(POLICY CMP0071)
-  cmake_policy(SET CMP0071 NEW)
-endif()
-
-find_package(CGAL REQUIRED OPTIONAL_COMPONENTS Core Qt5)
+find_package(CGAL REQUIRED OPTIONAL_COMPONENTS Core Qt6)
 find_package(LEDA QUIET)
 
-# Find Qt5 itself
-find_package(Qt5 QUIET COMPONENTS Widgets)
+# Find Qt6 itself
+find_package(Qt6 QUIET COMPONENTS Widgets)
 
-if(CGAL_Qt5_FOUND
-   AND Qt5_FOUND
+if(CGAL_Qt6_FOUND
+   AND Qt6_FOUND
    AND (CGAL_Core_FOUND OR LEDA_FOUND))
-  # ui files, created with Qt Designer
-  qt5_wrap_ui(UIS HDT2.ui)
 
-  qt5_add_resources(RESOURCE_FILES resources/Delaunay_triangulation_2.qrc)
+   set(CMAKE_AUTOUIC ON)
+   set(CMAKE_AUTORCC ON)
 
-  # cpp files
-  add_executable ( HDT2 HDT2.cpp ${CGAL_Qt5_RESOURCE_FILES} ${RESOURCE_FILES} ${UIS})
+  qt_add_executable ( HDT2 HDT2.cpp)
   target_include_directories(HDT2 PRIVATE ./ ./include)
```

```suggestion
  qt_add_executable ( HDT2 HDT2.cpp HDT2.ui resources/Delaunay_triangulation_2.qrc)
```

It seems the `.ui` is optional, because we can see the application working even without it.

But without the `.qrc`, the icons of the action buttons are empty. And the "About..." dialog is empty as well.

---

@SaillantNicolas I found out several such cases in the `CMakeLists.txt` you have modified. That should be fixed.